wandb/wandb_torch.py

```diff
@@ -104,7 +101,8 @@ def add_log_hooks_to_pytorch_module(
         if jupyter_run:
             self._jupyter_run = weakref.ref(jupyter_run)
 
-        module._wandb_hook_names = []
+        if not hasattr(module, "_wandb_hook_names"):
+            module._wandb_hook_names = []
```

Comment:
A couple of ideas to possibly make these assignments more robust:

This can be:
```
module.__dict__.setdefault("_wandb_hook_names", [])
```